config/production.json

```diff
@@ -119,6 +119,7 @@
 		"purchases/new-payment-methods": true,
 		"push-notifications": true,
 		"reader": true,
+		"reader/editor": false,
```

Comment:
Note that a feature flag is always `false` by default, unless specifically set to `true`. That being said, a `false` flag can just be omitted and it will work the same way.

Reply:
Good point--usually I err on the side of trying to be verbose, but I can see how the false flag values add clutter. Now removed.